velox/type/Timestamp.h

```diff
@@ -34,8 +35,20 @@ struct Timestamp {
  public:
   enum class Precision : int { kMilliseconds = 3, kNanoseconds = 9 };
   constexpr Timestamp() : seconds_(0), nanos_(0) {}
-  constexpr Timestamp(int64_t seconds, uint64_t nanos)
-      : seconds_(seconds), nanos_(nanos) {}
+  Timestamp(int64_t seconds, uint64_t nanos) {
+    VELOX_CHECK_GE(
+        seconds,
+        dwio::common::MIN_SECONDS,
+        "Expect seconds >= {} in Timestamp",
```

Comment:
No need to include this value in the message. VELOX_CHECK_GE will include it automatically.